t/live/jigsaw/chunk.t

```diff
@@ -1,37 +1,35 @@
-print "1..5\n";
-
 use strict;
+use warnings;
+use Test::More;
+
+use HTTP::Request;
 use LWP::UserAgent;
 
+plan tests => 10;
+
 my $ua = LWP::UserAgent->new(keep_alive => 1);
+isa_ok($ua, 'LWP::UserAgent', 'new: UserAgent instance');
 
 my $req = HTTP::Request->new(GET => "http://jigsaw.w3.org/HTTP/ChunkedScript");
+isa_ok($req, 'HTTP::Request', 'new: HTTP::Request instance');
```

Comment:
I realize this isn't your fault, but if there are tests that need the network, we should be using something like `Test::RequiresInternet`.